# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

MoveToStateActionGoal action_goal
MoveToStateActionResult action_result
MoveToStateActionFeedback action_feedback
